app/cloud/server/index.js

```diff
@@ -34,6 +37,22 @@ Meteor.startup(function() {
 			job: syncWorkspace,
 		});
 	});
+
+	const { workspaceRegistered } = retrieveRegistrationStatus();
+
+	if (process.env.REG_TOKEN && process.env.REG_TOKEN !== '' && !workspaceRegistered) {
+		try {
+			SystemLogger.info('REG_TOKEN Provided. Attempting to register');
+
+			if (!connectWorkspace(process.env.REG_TOKEN)) {
+				throw new Error('Couldn\'t register with token.  Please make sure token is valid or hasn\'t already been used');
+			}
+
+			console.log('Successfully registered with token provided by REG_TOKEN!');
```

Comment:
Use SystemLogger here

Reply:
Which variation?  I want this one to always log

Reply:
On the default log level we only log errors. The next level is `SystemLogger.info`

Reply:
So then console.log is the only option for logging 100% of the time without the log being a false error?

Reply:
Definitely seems this way